apim/3.x/templates/_helpers.tpl

```diff
@@ -299,3 +299,37 @@ Usage:
 {{- end }}
 {{- end }}
 {{- end }}
+
+{{- define "ui.base_href.defined" -}}
+{{- if contains "CONSOLE_BASE_HREF" (.Values.ui.env | toString) -}}
+{{- print "true" -}}
+{{ else }}
+{{- print "false" -}}
+{{- end }}
+{{- end }}
+
+{{- define "ui.base_href" -}}
+{{- if eq .Values.ui.ingress.path "/" }}
+{{- print "/" -}}
+{{ else}}
+{{- $name := regexFind "\\w+" .Values.ui.ingress.path -}}
+{{- print "/" $name "/" -}}
+{{- end }}
+{{- end }}
+
+{{- define "portal.base_href.defined" -}}
+{{- if contains "PORTAL_BASE_HREF" (.Values.portal.env | toString) -}}
+{{- print "true" -}}
+{{ else }}
+{{- print "false" -}}
+{{- end }}
+{{- end }}
+
+{{- define "portal.base_href" -}}
+{{- if eq .Values.portal.ingress.path "/" }}
+{{- print "/" -}}
+{{ else}}
+{{- $name := regexFind "\\w+" .Values.portal.ingress.path -}}
+{{- print "/" $name -}}
```

Comment:
No trailing "/" on that one? 

I mean in comparison of `{{- print "/" $name "/" -}}` L316

Reply:
yes, i don't know why but the console work only when CONSOLE_BASE_HREF=/xxxxxxx/ and the portal PORTAL_BASE_HREF=/xxxxx

Reply:
Yeah I was surprised to have the portal working without the `/` 🙈